tests/images.rs

```diff
@@ -363,7 +356,8 @@ fn postgres_one_plus_one_with_custom_mapped_port() {
 #[test]
 fn postgres_custom_version() {
     let docker = clients::Cli::default();
-    let postgres_image = images::postgres::Postgres::default().with_version(13);
+    let postgres_image =
+        RunnableImage::<images::postgres::Postgres>::default().with_tag("13-alpine");
```

Comment:
I just thought we can have an example test that shows such invocations are possible too. But not sure if this would be preferred by users? WDYT?

Reply:
I am having 2nd thoughts on the `Default` impl :grimacing: 

It doesn't actually save you from typing any characters because you still need to add the concrete image type. At that point, you might as well call `images::postgres::Postgres::default()`.

If we add the extension trait, this just becomes:

```rust
let postgres_image = images::postgres::Postgres::default().with_tag("13-alpine");
```

I think I would rather have just one idiomatic way of doing things and remove the `Default` impl again :sweat_smile:
Sorry for sending you on this wild goose chase!

Reply:
> It doesn't actually save you from typing any characters because you still need to add the concrete image type

Yeah. I felt the same. 😆 

> I think I would rather have just one idiomatic way of doing things and remove the Default impl again 😅
Sorry for sending you on this wild goose chase!

Ah no worries. It was nice to think about and try them and to have these discussions now 😄